.mocharc.json

```diff
@@ -1,5 +1,16 @@
 {
   "extension": ["ts"],
   "spec": "test/**/*.spec.ts",
-  "require": "sucrase/register"
+  "require": "sucrase/register",
+  "colors": true,
+  "reporter":"mocha-multi-reporters",
+  "reporterOption": "configFile=.mocharc.json",
+  "reporterEnabled": "mochawesome,mocha-junit-reporter",
+    "mochaJunitReporterReporterOptions": {
+        "mochaFile": "./coverage/junit.xml"
+    },
+    "mochawesomeReporterOptions": {
+      "reportDir": "coverage",
+      "reportName": "mochawesome"
+    }
```

Comment:
Just curious, what does mochawesome do other than provide a GUI version of the report?

Reply:
What do you mean "other then"?
Have you seen how pretty the test report is?
Other then it now be in dark mode (which I'm sure I can get it to work), it's perfection :) 

![Screen Shot 2022-01-05 at 23 13 47](https://user-images.githubusercontent.com/4012041/148343852-e505f7b6-9fec-45d6-9c47-de3442e015a2.png)

![Screen Shot 2022-01-05 at 23 13 30](https://user-images.githubusercontent.com/4012041/148343848-69864da2-86da-462e-8b66-9b0ac3b4b28f.png)

![Screen Shot 2022-01-05 at 23 13 23](https://user-images.githubusercontent.com/4012041/148343845-4285cd5a-28e5-47f3-a1ca-9b3056890999.png)